crossing-training.lic

```diff
@@ -382,27 +382,51 @@ class CrossingTraining
   def calculate_mana(min, more, discern_data, cyclic)
     total = min + more
     total = (total * @settings.prep_scaling_factor).floor
-    discern_data['mana'] = [(total / 4.0).ceil, min].max
+    discern_data['mana'] = [(total / 5.0).ceil, min].max
     remaining = total - discern_data['mana']
-    if remaining > @settings.cambrinth_cap
-      discern_data['mana'] = discern_data['mana'] + (remaining - @settings.cambrinth_cap)
+    if @settings.cambrinth_items[0]['name']
+      cambrinth_items = @settings.cambrinth_items
+    else
+      cambrinth_items = []
+      cambrinth_items[0] = {}
+      cambrinth_items[0]['name'], cambrinth_items[0]['cap'], cambrinth_items[0]['stored'] = map_cambrinth(@settings)
+    end
+    total_cambrinth_cap = 0
+    cambrinth_items.each do |item|
+      total_cambrinth_cap += item['cap']
+    end
```

Comment:
You can just assign directly to total cap with `[{'cap' => 5},{'cap'=>14}].map{|x| x['cap']}.inject(&:+)`

Reply:
Wasn't aware of this bit of code, thanks.  Updated.